wpilib/wpilib/driverstation.py

```diff
@@ -388,12 +424,11 @@ def getJoystickIsXbox(self, stick: int) -> bool:
         if stick < 0 or stick >= self.kJoystickPorts:
             raise IndexError("Joystick index is out of range, should be 0-%s" % self.kJoystickPorts)
 
-        with self.cacheDataMutex:
-            # TODO: Remove this when calling for descriptor on empty stick no longer crashes.
-            if 1 > self.joystickButtons[stick].count and 1 > len(self.joystickAxes[stick]):
-                self._reportJoystickUnpluggedWarning("WARNING: Joystick on port {} not avaliable, check if controller is "
-                                                     "plugged in.\n".format(stick))
-                return False
+        # TODO: Remove this when calling for descriptor on empty stick no longer crashes.
+        if 1 > self.joystickButtons[stick].count and 1 > len(self.joystickAxes[stick]):
```

Comment:
Neither of these checks are present upstream anymore. Presumably it doesn't crash anymore?